src/osd/osd_types.cc

```diff
@@ -5965,6 +5965,13 @@ SnapSet SnapSet::get_filtered(const pg_pool_t &pinfo) const
   return ss;
 }
 
+#ifdef WITH_SEASTAR
```

Comment:
Note: This can probably be used also by classic (In a separate PR), so it's worth the change to common files.